packages/app/src/components/LeftSidebar.tsx

```diff
@@ -83,7 +83,7 @@ export const LeftSidebar: FC<{
       key={project.metadata.id}
     >
       <div className="toggle-tab" onClick={() => setSidebarOpen(!sidebarOpen)}>
-        {sidebarOpen ? <ExpandLeftIcon /> : <ExpandRightIcon />}
```

Comment:
?

Reply:
Happy to discuss in a separate PR, but this PR is already pretty big to review so let's keep it to just the jotai changes

Reply:
Fixed